website/src/main/tut/docs/rules/NoInfer.md

```diff
@@ -1,27 +1,51 @@
 ---
 layout: docs
 title: NoInfer
+tag: rule
```

Comment:
What does this do? Just curious.

Reply:
Not sure. I saw it in the Disable rule. I thought was there in all the rules

Reply:
Ahah woops, that's a failed attempt of mine when putting the new website together. It doesn't do anything, let's remove it in another PR :)